Items/T1/BitterRoot.cs

```diff
@@ -20,8 +20,8 @@ public class BitterRoot : Item<BitterRoot> {
         public float healthCap {get; private set;} = 3f;
 
         protected override string NewLangName(string langid = null) => displayName;        
-        protected override string NewLangPickup(string langid = null) => "Gain " + Pct(healthMult) + " max hp.";
-        protected override string NewLangDesc(string langid = null) => "Increases <style=cIsHealing>health</style> by <style=cIsHealing>" + Pct(healthMult) + "</style> <style=cStack>(+" +Pct(healthMult)+ " per stack, linear)</style>, up to a <style=cIsHealing>maximum</style> of <style=cIsHealing>+"+Pct(healthCap)+"</style>.";        
+        protected override string NewLangPickup(string langid = null) => "Gain " + Pct(healthMult) + " max health.";
```

Comment:
Use of "hp" vs "health" here is a stylistic choice for the pickup text, as the former matches with the RoR1 item's pickup text.

Reply:
Can we at least captitalize it from `hp` to `HP`? Still your call though.